examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -1,12 +1,11 @@
 # Create ParallelCluster
 
-Once your prerequisite infrastructure and AMI are all set up, you are ready to create a ParallelCluster. Copy the following content into a launch.yaml file in your local desktop where AWS ParallelCluster CLI is installed:
+1. Once your VPC is set up, you are ready to create a ParallelCluster. Copy the following content into a launch.yaml file in your local desktop where AWS ParallelCluster CLI is installed:
```

Comment:
VPC, Parallel Cluster python package, and key pair

Reply:
Will update in new commit